modules/installation-initializing-manual.adoc

```diff
@@ -263,13 +263,6 @@ Because of this behavior, you must always keep a backup of these files in case y
 
 endif::restricted,restricted-upi[]
 
-ifndef::aws-china,aws-gov,aws-secret,azure-gov,ash,ash-default,ash-network,gcp-shared,ibm-cloud-private,ibm-power-vs-private,ibm-cloud-restricted[]
```

Comment:
Because this module is included in several other assemblies (such as `installing-gcp-private.adoc` and several [vsphere assemblies](https://docs.openshift.com/container-platform/4.15/installing/installing_vsphere/upi/installing-vsphere.html#installation-initializing-manual_installing-vsphere)) I am worried that completely removing this `ifndef` is too broad. 

I recommend adding an `ifeval` for the two bare metal assemblies that include this module, and adding them to the condition on line 266/272 to exclude the note from those assemblies. What do you think?

Reply:
@bscott-rh I pondered that, but I believe what the NOTE box says has become meaningless. It tries to muddy the surrounding text in all install guides. Does it preserve something needed in installing-gcp-private.adoc?

Reply:
hmmm I think if we are unsure about the impact of this note, maybe the thing to do is get QE to give a :+1:/:-1: on if it really goes in any assembly it's currently in?

Reply:
Fair point, I'm OK with you removing it. The doc is easier to read without it

Reply:
@jeana-redhat I've got the original QE person looking at the JIRA right now.

Reply:
@achuzhoy your QE review needed, please!

Reply:
Yeah. Looks cleaner without.